app/assets/stylesheets/basic.scss

```diff
@@ -1 +1,27 @@
+#background {
+  background: url("http://www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png")
```

Comment:
URL `http://www.onefivesports.com/wp-content/uploads/2012/06/basket_ball_court1.png` should not contain protocol or domain